daemon/src/projection.rs

```diff
@@ -729,6 +729,28 @@ impl Cfd {
 
         Some(url)
     }
+
+    fn is_open(&self) -> bool {
```

Comment:
🛠 This commit seems to introduce a function that is not used in code, but only in the tests. Is this commit incomplete or is this just to showcase what you have planned?

Reply:
I tried to explain it in the commit message, not sure if you saw it: `This will come handy later on when creating metrics.` 
Meaning, this is a separate commit introducing this function which will be used for the metrics later.